hardware/EuroPi/build_guide.md

```diff
@@ -192,6 +192,20 @@ The 'front' of the Jack PCB is the side with the OLED, jack, and button outlines
 |-----|----|
 |![DSC2406](https://user-images.githubusercontent.com/79809962/148679028-98e41fce-fe47-40d5-b833-9d615c923592.jpg)|![DSC2407](https://user-images.githubusercontent.com/79809962/148679039-b93381e4-b30d-47a6-935f-a1ab66642e18.jpg)|
 
+#### Required Components for Jack PCB
+|Component|Quantity|
+|---------|--------|
+|1k Resistor|8|
+|4.7k Resistor|6|
+|100nF Capacitor|2|
+|LEDs|6|
+|4x2 Male Header|2|
+|Jumper Wires|4|
```

Comment:
Plurals written as singular in the Pico PCB table, very minor nitpick but might as well keep it consistent!

Reply:
Done.